### ISSUE TITLE
Attempt to Rename Database in OE Failed

### ISSUE THREAD

**Issue by chlafreniere** [microsoft/azuredatastudio#22866]
(Latest insiders, Windows)

Right click on a database -> Rename.
Expected: rename succeeds
Actual: Get error message in tasks pane:

```
Rename database 'Northwind' to 'Northwind2': An error occurred while renaming database 'Northwind' to 'Northwind2'. Error parsing message contents
...
Error converting value "Database" to type 'Microsoft.SqlTools.ServiceLayer.ObjectManagement.SqlObjectType'. Path 'params.objectType', line 1, position 573. ---> Requested value 'Database' was not found.
```

(note: connected to local docker sql instance)

**Comment by cheenamalhotra** [comment]
I see renaming 'Database' is probably not supported by Object Management Service.. 

@alanrenmsft can you confirm? Is it something that SMO supports?
If not, we should probably remove 'Rename' operation from 'Database' objects.

**Comment by aasimkhan30** [comment]
Fixed with #2030
